pyproject.toml

```diff
@@ -16,24 +14,20 @@ classifiers = [
     "Natural Language :: English",
     "Programming Language :: Python :: 3.11",
 ]
-requires-python = ">=3.11"
-
-dependencies = [
-    "matplotlib>=3.6",
-    "pandas>2.1.4",
-    "scipy>1.10",
-    "tqdm>4.50",
-]
 
-[project.urls]
+[tool.poetry.urls]
 "Bug Tracker" = "https://github.com/equinor/completor/issues"
 
 [tool.poetry.dependencies]
+python = "^3.11"
+matplotlib = "^3.6"
+numpy = "<2"
```

Comment:
Why pin numpy to less than 2? Is there a specific reason for this?
Also, this will become a problem once komodo starts building with numpy2.